src/Repository/Formation/AxeRepository.php

```diff
@@ -16,11 +16,11 @@ public function __construct(RegistryInterface $registry)
     /**
      * @return Axe[]
      */
-    public function findAllWithArticles(): array
+    public function findAllWithModules(): array
     {
         return $this->createQueryBuilder('a')
-            ->addSelect('articles')
-            ->leftJoin('a.articles', 'articles')
+            ->addSelect('modules')
+            ->leftJoin('a.modules', 'modules')
```

Comment:
if the method name is `...WithModules` it should be `innerJoin` here, no?

Reply:
yes, could be better to not show empty `axes` and empty `paths` 👍